starboard/nplb/posix_compliance/posix_string_compare_no_case_n_test.cc

```diff
@@ -0,0 +1,61 @@
+// Copyright 2015 The Cobalt Authors. All Rights Reserved.
```

Comment:
It yells about files having the same name - although I realize that's because I forgot to add a posix_compliance namespace to them.  Should I add the namespace & rename?